### PR TITLE
Add Stephen Judes blog

### PR DESCRIPTION
**Pull request by prondubuisi**
#### What does this PR do?

Add Stephen Jude's blog

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +117,7 @@ Useful articles and learning resources.
 - [Free Code Camp](https://www.freecodecamp.org) - Software development related posts
 - [CSS Tricks](https://css-tricks.com) - CSS related posts
 - [DEV.to](https://dev.to) - The All in One Blogs Portal for Developers
+- [Stephen Jude's Blog](http://stephenjude.me/) - Everything about PHP, Laravel and
```
Comment (BolajiAyodeji) [inline README.md:120]: Thanks for this amazing PR!

Can confirm that the description is complete?

Comment (prondubuisi) [reply README.md:120]: No Chief @BolajiAyodeji it isn't. let me update it

Comment (prondubuisi) [reply README.md]: Updated @BolajiAyodeji

**Comment by prondubuisi** [comment]
Fixed merge conflicts @BolajiAyodeji

**Pull request merged**